src/styles.scss

```diff
@@ -3,3 +3,58 @@
 body {
   margin: 0;
 }
+
+.center {
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  margin: auto;
+}
+
+.left {
+  justify-content: left;
+  text-align: left;
+  align-items: left;
+  margin: auto;
+}
+
+.login {
+  display: grid;
+  grid-template-columns: 1/3 1/3 1/3;
+
+  .subtitle {
+    font-size: 18px;
+    font-family: "Silkscreen", cursive;
+    margin: 1;
+    white-space: normal;
+    display: block;
+  }
+
+  .text {
+    font-size: 14px;
+    font-family: "Silkscreen", cursive;
+    margin: 1;
+    white-space: normal;
+  }
+
+  .label {
+    align-content: start;
+    text-align: start;
+  }
+
+  .button {
+    margin-top: 2rem;
+  }
+
+  .title {
+    font-size: 36px;
+    font-family: "Silkscreen", cursive;
+    margin: 1;
+    white-space: normal;
+    display: block;
+  }
+
+  .link {
+    margin-bottom: 0px;
+  }
+}
```

Comment:
component-specific styles should not be added to global styles.scss -> move to specific components